src/components/Projects/HoldingsProjects.tsx

```diff
@@ -34,8 +34,8 @@ export default function HoldingsProjects() {
           {projects && projects.length > 0 && (
             <React.Fragment>
               <Grid>
-                {projects.map(p => (
-                  <ProjectCard project={p} />
+                {projects.map((p, i) => (
+                  <ProjectCard key={i} project={p} />
```

Comment:
I am pretty sure from memory that this is a antipattern for react and should use a unique key id, rather than index.

Reply:
yep totally is, was carryover from the old PR 🙏 . Thanks for catching!!